Cargo.toml

```diff
@@ -24,9 +25,5 @@ pyo3 = { version = "0.15", default-features = false }
 [dev-dependencies]
 pyo3 = { version = "0.15", features = ["auto-initialize"] }
 
-[features]
-default = []
-rayon = ["ndarray/rayon"]
```

Comment:
Might be worth noting this removal in the CHANGELOG. (Alongside a suggestion to just use ndarrays feature directly).